accepted/future-releases/constructor-tearoffs/feature-specification.md

```diff
@@ -402,6 +402,18 @@ A compile-time error occurs if a constructor tear-off denotes a generative const
 
 *A generative constructor declared in an abstract class may well be executed as part of the execution of a constructor tear-off, via a superinitializer. It cannot, however, be torn off on its own, because the execution of a constructor tear-off is an instance creation, and abstract classes do not have instances. There is no similar constraint on a factory constructor.*
 
+### Compile-time errors
+
+This section lists the compile-time errors associated with this feature. The _italic_ entries are commentary, which indicates that they are stated elsewhere already.
+
```

Comment:
I'd say "summarizes" instead of "lists", since it only lists *one* entry, then the rest are then stated elsewhere. (A "list of one thing" is just a thing).

Also, it doesn't really summarize all the errors.